src/components/List/List.js

```diff
@@ -18,29 +19,46 @@ export const timeDifference = (timestampInMilli) => {
   const difference = date - timestampInMilli;
   const comparison = difference < dayInMilli;
 
-  console.log(`differience in ms = ${difference} = ${date} - ${timestampInMilli}`);
-  console.log(`less than 24 hours = ${comparison} = ${difference} < ${dayInMilli}`);
+  // console.log(`difference in ms = ${difference} = ${date} - ${timestampInMilli}`);
+  // console.log(`less than 24 hours = ${comparison} = ${difference} < ${dayInMilli}`);
 
   // this will return true when its been less than 24 hours
   // false otherwise
   return comparison;
 };
 
+
 const List = () => {
   const { token } = useParams();
   const [docs, setDocs] = useState([]);
+  const [userSearch, setUserSearch] = useState('');
 
   async function checkboxChange(checked, id) {
     // three different possible states
-    // 1: purchaseDate is null, checkbox is unchecked => checking runs servertimestamp
-    // 2: purchaseDate exists, checkbox is checked => unchecking sets purchaseDate to null
-    // 3: purchaseDate exists, checkbox is unchecked => checking runs serverTimestamp
+    // 1) purchaseDate is null, checkbox is unchecked => checking runs servertimestamp
+    // 2) purchaseDate exists, checkbox is checked => unchecking sets purchaseDate to null
+    // 3) purchaseDate exists, checkbox is unchecked => checking runs serverTimestamp
 
     await updateDoc(doc(db, token, id), {
       purchaseDate: checked ? null : serverTimestamp(),
     });
   }
 
```

Comment:
hmm - can this be deleted here? This must be code for issue 10, right? I'm anticipating that keeping these lines commented out here could create a merge conflict if this one is merged in as-is, with the un-commented code in the other PR.